sycl/test/array_param/array-kernel-param-nested-run.cpp

```diff
@@ -0,0 +1,135 @@
+// This test checks kernel execution with array parameters inside structs.
+
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// XFAIL: *
```

Comment:
Is this one covered by a github issue?

Reply:
Not directly, but the tests marked XFAIL are tests with nested arrays. Nested arrays will only work once struct decomposition is implemented. This is being handled in PR#1877.  Maybe we can add a new Github issue for nested arrays (expected to be fixed by PR#1877) to make this all clearer.

Reply:
I created this issue - https://github.com/intel/llvm/issues/2018.